src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -4438,13 +4488,25 @@ VOID UnwindManagedExceptionPass2(PAL_SEHException& ex, CONTEXT* unwindStartConte
                 // We also need to reset the scanned stack range since the scanned frames will be
                 // obsolete after the unwind of the native frames completes.
                 ExceptionTracker* pTracker = GetThread()->GetExceptionState()->GetCurrentExceptionTracker();
-                pTracker->CleanupBeforeNativeFramesUnwind();
+                if (pTracker != NULL)
```

Comment:
Still investigating this. So far, I've been able to confirm the `ExceptionTracker` instance is being NULL'd out during the last call to `ProcessCLRException` in `UnwindManagedExceptionPass2`.

Reply:
@janvorli I think this is actually okay. The `ExceptionTracker` instance is being cleaned up in `ProcessCLRException` because of the "invalid Reverse P/Invoke" logic at 

https://github.com/dotnet/runtime/blob/82c917aa6119c553c4fa093265c52f34476e7e13/src/coreclr/vm/exceptionhandling.cpp#L1229-L1235

I believe this is the cleanup we want. Should I assert something in this area?

Reply:
Added comment as to why this is okay.